ios/Podfile

```
@@ -0,0 +1,40 @@
+
+  target 'UBIFastPassTests' do
+    inherit! :complete
+    # Pods for testing
+  end
+
+  post_install do |installer|
+    # https://github.com/facebook/react-native/blob/main/packages/react-native/scripts/react_native_pods.rb#L197-L202
+    react_native_post_install(
+      installer,
+      config[:reactNativePath],
+      :mac_catalyst_enabled => false,
```

rubocop

[convention] Avoid comma after the last parameter of a method call.

(Style/TrailingCommaInArguments)